### ISSUE TITLE
How can i turn off the led light after the computer been shut down? (Adalight)

### ISSUE THREAD

**Issue by gu1chen** [psieg/Lightpack#590]
The light keeps on in a single color after i shut down my computer, it really bothers me in days.
I have to turn off the power supply additionally to make sure the lights off with computer at the same time....
All the function is working great, i lreally love the screen grabing, thanks for your job
Can you help with my issues?
Attach with my settiungs
![屏幕截图 2023-05-28 214935](https://github.com/psieg/Lightpack/assets/134853219/7ccf0cf2-4fb4-40c3-9005-b312b5bcb053)
![屏幕截图 2023-05-28 214922](https://github.com/psieg/Lightpack/assets/134853219/ebdd428a-c6e3-4e0a-9614-f78baa1322dd)
![屏幕截图 2023-05-28 214853](https://github.com/psieg/Lightpack/assets/134853219/5924e66b-3102-4d94-a26e-b647a5668a9b)

**Comment by Exioncore** [comment]
![image](https://github.com/psieg/Lightpack/assets/5459344/20255784-d942-44a3-b82c-5097934b72c0)
Making sure all those "Keep lights ON after ..." are unticked should make them turn off on shutdown? 
Personally though, I configured my Adalight firmware to shut off the lights after 5 seconds of no signal.

**Comment by gu1chen** [comment]
Yeah...I have tried to turn off those "keep lights on"s，but it doesn't help at all. 
The light will just back to life after 3 or 5 seconds when there's no signal receiving from screen.
How to configue by the way？That sounds like the right  solution🤔

**Comment by zomfg** [comment]
from your description sounds like that's the way your device/firmware handles the no-signal situation
I'd look for alternative firmwares or a way to configure the current one

**Comment by HandsomGhost** [comment]
> ![image](https://user-images.githubusercontent.com/5459344/241589089-20255784-d942-44a3-b82c-5097934b72c0.png) Making sure all those "Keep lights ON after ..." are unticked should make them turn off on shutdown? Personally though, I configured my Adalight firmware to shut off the lights after 5 seconds of no signal.

You mind sharing how you configured the Adalight firmware to shut off the lights after 5 seconds of no signal. I'm having the same problem, thanks.

**Comment by Exioncore** [comment]
> > ![image](https://user-images.githubusercontent.com/5459344/241589089-20255784-d942-44a3-b82c-5097934b72c0.png) Making sure all those "Keep lights ON after ..." are unticked should make them turn off on shutdown? Personally though, I configured my Adalight firmware to shut off the lights after 5 seconds of no signal.
> 
> You mind sharing how you configured the Adalight firmware to shut off the lights after 5 seconds of no signal or the firmware. I'm having the same problem, thanks.

This is a version of the Adalight firmware: https://github.com/dmadison/Adalight-FastLED/blob/master/Arduino/LEDstream_FastLED/LEDstream_FastLED.ino

The SerialTimeout value determines how many seconds without serial data need to elapse before turning the lights off. You will also need to configure correctly the Num_Leds, LED_TYPE, COLOR_ORDER and PIN_DATA.

**Comment by gu1chen** [comment]
it's weird but after changed my graphic card from RTX3080 to RX7900XTX，this issue fixed...I don't know why but It's happened...



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 10, 2023 05:14 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [psieg/Lightpack] How can i turn off the led light after thecomputer been shut down? (Adalight) (Issue #590)




   
 Making sure all those "Keep lights ON after ..." are unticked should make them turn off on shutdown? Personally though, I configured my Adalight firmware to shut off the lights after 5 seconds of no signal.
  
You mind sharing how you configured the Adalight firmware to shut off the lights after 5 seconds of no signal or the firmware. I'm having the same problem, thanks.
  
This is a version of the Adalight firmware: https://github.com/dmadison/Adalight-FastLED/blob/master/Arduino/LEDstream_FastLED/LEDstream_FastLED.ino
 
The SerialTimeout value determines how many seconds without serial data need to elapse before turning the lights off. You will also need to configure correctly the Num_Leds, LED_TYPE, COLOR_ORDER and PIN_DATA.

**Comment by HandsomGhost** [comment]
> > > ![image](https://user-images.githubusercontent.com/5459344/241589089-20255784-d942-44a3-b82c-5097934b72c0.png) Making sure all those "Keep lights ON after ..." are unticked should make them turn off on shutdown? Personally though, I configured my Adalight firmware to shut off the lights after 5 seconds of no signal.
> > 
> > 
> > You mind sharing how you configured the Adalight firmware to shut off the lights after 5 seconds of no signal or the firmware. I'm having the same problem, thanks.
> 
> This is a version of the Adalight firmware: https://github.com/dmadison/Adalight-FastLED/blob/master/Arduino/LEDstream_FastLED/LEDstream_FastLED.ino
> 
> The SerialTimeout value determines how many seconds without serial data need to elapse before turning the lights off. You will also need to configure correctly the Num_Leds, LED_TYPE, COLOR_ORDER and PIN_DATA.

How would you install? I was looking on the FastLED Github and not too sure. Thanks for the respond

**Comment by and7ey** [comment]
I have the same problem. Not sure if it is possible to update the firmware on my box (does it have arduino inside?)

![Image](https://github.com/user-attachments/assets/2bff3a1f-c103-4785-9c4d-298c5390ea2d)
